<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamlineConfig</class>
 <widget class="QWidget" name="StreamlineConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>923</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="8" column="0" colspan="3">
      <widget class="QGroupBox" name="configRuledSurfaceGroupBox">
       <property name="title">
        <string>Config Ruled Surface</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="3" column="0">
           <widget class="QCheckBox" name="closeSurfaceCheckBox">
            <property name="text">
             <string>Close Surface</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Distance Factor</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="distanceFactorSpinBox"/>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="onRatioSpinBox"/>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="offsetSpinBox"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Offset</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="passLinesCheckBox">
            <property name="text">
             <string>Pass Lines</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Ruled Mode</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Resolution</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="ruledModeCombo">
            <item>
             <property name="text">
              <string>Resample</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Point Walk</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="resolutionSpinBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>OnRatio</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="orientLoopsCheckBox">
            <property name="text">
             <string>Orient Loops</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Display Effect</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>Open ...</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QComboBox" name="integrationDirectionCombo">
       <item>
        <property name="text">
         <string>FORWARD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BACKWARD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BOTH</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Integration Direction</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Integrator Type</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QComboBox" name="integratorTypeCombo">
       <item>
        <property name="text">
         <string>RUNGE_KUTTA2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RUNGE_KUTTA4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RUNGE_KUTTA45</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NONE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UNKNOWN</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fileEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="QGroupBox" name="configLineGroupBox">
       <property name="title">
        <string>Config Line</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="point2YSpinBox">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="point1YSpinBox">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Point1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="point1XSpinBox">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Point2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="point2XSpinBox">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QDoubleSpinBox" name="point1ZSpinBox">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QDoubleSpinBox" name="point2ZSpinBox">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QComboBox" name="displayModeCombo">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tube</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Surface</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Display Mode</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="2">
      <widget class="QComboBox" name="displayEffectCombo">
       <item>
        <property name="text">
         <string>Transparent</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Opaque</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wireframe</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QGroupBox" name="configPointGroupBox">
       <property name="title">
        <string>Config Point</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Center</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="centerXSpinBox">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="centerYSpinBox">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QDoubleSpinBox" name="centerZSpinBox">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2" colspan="2">
           <widget class="QSpinBox" name="numOfPointsSpinBox"/>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Number of Points</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Radius</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2" colspan="2">
           <widget class="QDoubleSpinBox" name="radiusSpinBox">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>File:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="3">
      <widget class="QGroupBox" name="configTubeGroupBox">
       <property name="title">
        <string>Config Tube</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="1">
           <widget class="QSpinBox" name="numOfSidesSpinBox"/>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="radiusFactorSpinBox"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Number of Sides</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Radius Factor</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="cappingCheckBox">
            <property name="text">
             <string>Capping</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Source</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="sourceCombo">
       <item>
        <property name="text">
         <string>Point</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Line</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="12" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
